spec/driver_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+describe PiPiper::Driver do
+    it 'has a version number' do
+      expect(PiPiper::Driver::VERSION).not_to be nil
+    end
+
+    it 'allow some part of the API' do
+      expect(PiPiper::Driver::ALLOW).to eq []
```

Comment:
As stated above, I don't think this is actually necessary. The public methods exposed by `Driver` should be enough.